### PR TITLE
Edit smart allocation responsive

### PR DESCRIPTION
**Pull request by almujtabamohammed**

### CODE REVIEW

File: src/components/shared/pageLoader/pageLoader.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Col } from "../layout/flex";
 import LoadingSpinner from "../loading-spinner/loading-spinner";
 
 const PageLoader: FC = () => (
-    <Col className="w-full h-full bg-black-2 fixed z-30 left-0 top-0 items-center justify-center">
+    <Col className="w-full h-[calc(100vh_-_7rem)] md:h-[calc(100vh_-_7rem_-_60px)] items-center justify-center">
```
Comment (ameer-greenbull) [inline src/components/shared/pageLoader/pageLoader.tsx:6]: suggestion why not use dialog component from radix?

**Review by ameer-greenbull — APPROVED** [review]

File: src/components/app/smart-allocation/authed-smart-allocation/smart-allocation-tabs/smart-allocation-holdings-tab/smart-allocation-holdings-tab.tsx [hunk 2]
```diff
@@ -32,77 +32,127 @@ const SmartAllocationHoldingsTab: FC<{ smartAllocationHoldings: SmartAllocationA
         )
     }, [smartAllocationHoldings, smartAllocationTotalEvaluation])
 
+    const tableHeader = useMemo(() => {
+        if (isTabletOrMobileScreen) {
+            return (
+                <thead>
+                    <tr>
```
Comment (AbdallaEmadEldinAbdalla) [inline src/components/app/smart-allocation/authed-smart-allocation/smart-allocation-tabs/smart-allocation-holdings-tab/smart-allocation-holdings-tab.tsx:39]: Why we don't create 2 arrays and map to make it more clear

File: src/components/app/smart-allocation/authed-smart-allocation/smart-allocation-tabs/smart-allocation-holdings-tab/smart-allocation-holdings-tab.tsx [hunk 3]
```diff
@@ -32,77 +32,127 @@ const SmartAllocationHoldingsTab: FC<{ smartAllocationHoldings: SmartAllocationA
...
+                        <tr key={asset.name}>
+                            <td>
+                                <Col className="gap-3">
+                                    <p>{asset.asset_details?.asset_data?.name}</p>
+                                    <span className="text-sm text-grey-1">{asset.name}</span>
+                                </Col>
+                            </td>
+                            <td>
+                                <Col className="gap-3">
+                                    <p>{formatNumber(asset?.available ?? 0)}</p>
```
Comment (AbdallaEmadEldinAbdalla) [inline src/components/app/smart-allocation/authed-smart-allocation/smart-allocation-tabs/smart-allocation-holdings-tab/smart-allocation-holdings-tab.tsx:81]: We don't need text styling here?

**Review by AbdallaEmadEldinAbdalla — CHANGES_REQUESTED** [review]
Why most of th `p` tags doesn't have any styling?

**Pull request merged**